fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/JDBCConstants.java

```diff
@@ -47,6 +47,11 @@
     public static final String IS_DELETED_NO = "IS_DELETED = 'N'";
     public static final String IS_DELETED = "IS_DELETED";
 
+    // Special parameter names
+    public static final String PARAM_NAME_PROFILE = "_profile";
+    public static final String PARAM_NAME_SECURITY = "_security";
+    public static final String PARAM_NAME_TAG = "_tag";
```

Comment:
note to self:  double-check that we have `_source` here as well after we merge mike's changes in

Reply:
these were moved to SearchConstants and I confirmed that it includes `_source`